build/optaplanner-ide-config/pom.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-parent</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>optaplanner-ide-config</artifactId>
+  <name>OptaPlanner IDE Configuration</name>
```

Comment:
Typically we add `<description>` and `<url>` too.
The description section is mostly duplicate indeed, so that's an issue in itself.
And the url is because maven used to have some crazy rules that made it append the artifactId to inherited url's for maven site, which is long since death, in the manifest.mf of the resulting jar IIRC. I am not sure if that's still the case. 

Either way, if we drop description and/or url overwritting here, we should do that on all child poms at the same time, so it's a separate issue.

Reply:
Let's add the description and url for now to keep it consistent with other modules.